sql/2021/mobile-web/navigator_device_memory_usage.sql

```diff
@@ -0,0 +1,13 @@
+#standardSQL
+# Usage of Navigator.deviceMemory
+SELECT
+  client,
+  feature,
+  num_urls AS freq,
+  total_urls AS total,
+  pct_urls AS pct
+FROM
+  `httparchive.blink_features.usage`
+WHERE
+  yyyymmdd = '20210701' AND
+  feature = 'NavigatorDeviceMemory'
```

Comment:
Same comment about whether we need to `OR` with the feature `id`? Again, not sure we do to be honest...